### ISSUE TITLE
Instructions for nav logos

### ISSUE THREAD

**Issue by kayaelle** [learningeconomy/admin-dashboard#41]
What size should the logo be for the nav? Is there a different logo image for expanded versus compressed? 

Do we want to move this into an admin form like the create users form so that non-technical users can just upload their logo(s) and it works?

**Comment by goblincore** [comment]
@kayaelle Yeah that would be fairly trivial to add the ability for the user to do that, although if we are going to start storing uploaded files might have to think/talk about that
